src/js/AccessibilityUtils.js

```diff
@@ -13,9 +13,10 @@
 // limitations under the License.
 
 goog.require('axs.browserUtils');
+goog.require('axs.color');
+goog.require('axs.color.Color');
```

Comment:
Will this be automatically imported by the line above? Not super familiar with closure compiler.